components/Mailer/basic_usage.rst

```diff
@@ -0,0 +1,193 @@
+Basic usage
+===========
+
+.. _component_mailer_provider_default-settings-provider:
+
+DefaultSettingsProvider
+-----------------------
+
+The **DefaultSettingsProvider** is simple model of default settings for **Sender**.
```

Comment:
It is not a model, but a service, which provides you with default sender address and sender name.